baseband/vlbi_base/encoding.py

```diff
@@ -11,19 +11,25 @@
 
 # The high mag value for 2-bit reconstruction.
 OPTIMAL_2BIT_HIGH = 3.3359
-r"""Optimal high value for a 2-bit digitizer for which the low value is 1.
+r"""Optimal high value for a 2-bit digitizer for which the low value is
++/- 1.
 
-It is chosen such that for a normal distribution in which 68.269% of all values
-are at the low level, this is the mean of the others, i.e.,
+It is chosen such that for a normal distribution in which 68.269% of all
+values are at the low level, this is the mean of the others, i.e.,
 
 .. math::
 
-     l = \frac{\int_1^\infty x \exp(-\frac12x^2) dx}
-              {\int_0^1 x \exp(-\frac12x^2) dx} \times
-         \frac{\int_0^1 \exp(-\frac12x^2)dx}
-              {\int_1^\infty \exp(-\frac12x^2) dx}
+     l = \frac{\int_\sigma^\infty x \exp(-\frac{x^2}{2\sigma^2}) dx}
+              {\int_\sigma^\infty \exp(-\frac{x^2}{2\sigma^2}) dx},
+
+where the standard deviation :math:`\sigma = 2.1745`, which can
+implicitly be determined from:
+
+.. math::
+
+     1 = \frac{\int_0^\sigma x \exp(-\frac{x^2}{2\sigma^2}) dx}
+          {\int_0^\sigma \exp(-\frac{x^2}{2\sigma^2}) dx}.
```

Comment:
This is solvable, so maybe just add that:
```
sigma = \frac{\sqrt{\pi/2}\erf(\sqrt{1/2})}{1-1/\sqrt{e}} = 2.174564
```
and then the high level is
```
l = \frac{1}{(\sqrt{e}-1)*(1/\erf(\sqrt{1/2}) - 1) = 3.316505
```

Reply:
Agreed that it's solvable, and I also obtained `sigma =  `l = 3.3165047…`, but   `OPTIMAL_2BIT_HIGH = 3.3359` in the code, so just want to confirm that I should change that to `3.3165`.  Also `TWO_BIT_1_SIGMA = 2.1745` rather than `2.1746`, but changing this shouldn't break anything.